### PR TITLE
Phase 3 updates for ACI plugin

### PR DESCRIPTION
**Pull request by sunithasomanna**
- Added section - Filing Resource Aggregator for ODIM defects
- Removed section - Configuring proxy server for a plugin version
- Minor updates to  ACI plugin deployment instructions - hookscript info added +saving the proxy configuration file install/templates/<pluginname>_proxy_server.conf.j2 to the ODIM plugin path + edited plugin configuration yaml file.
- Edited information about lbport
- Removed $ symbols from commands
- Added step to build ACI image
- Edited the command "cp PluginCiscoACI/install/Kubernetes/helmcharts/aciplugin.sh ~/plugins/aciplugin"
- Added "odimra-secret" step
- Updated "default ports" section

### CODE REVIEW

**Review by bharath-b-hpe — APPROVED** [review]

**Review by shivacharanms — APPROVED** [review]

**Review by jeevan-kamkar — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -179,37 +198,46 @@ Kubernetes cluster is set up and the resource aggregator is successfully deploye
...
+13. Move odimra_kafka_client.key, odimra_kafka_client.crt, odimra_server.key and odimra_server.crt stored in odimCertsPath to a different folder.
 
-13. Run the following command: 
+    <blockquote> NOTE: odimCertsPath is the absolute path of the directory where certificates required by the services of Resource Aggregator for ODIM are present. This parameter is configured in the `kube_deploy_nodes.yaml` file.</blockquote>
 
-        $ python3 odim-controller.py --config /home/${USER}/ODIM/odim-controller/scripts/kube_deploy_nodes.yaml --upgrade odimra-config
+14. Update odimra-secrets:
 
-14. Run the following command to install the Cisco ACI plugin: 
+       ```
```
Comment (sukhwinder20) [inline README.md:214]: cd ~/ODIM/odim-controller/scripts

Add this command

File: README.md [hunk 2]
```diff
@@ -39,44 +52,48 @@ Kubernetes cluster is set up and the resource aggregator is successfully deploye
...
+
+   3. ```
+      ./build_images.sh
+      ```
+
+4. On the deployment node, copy the Cisco ACI plugin configuration file and the hook script to `~/plugins/aciplugin`.
 
    ```
-   $ sudo chown odimra:odimra /var/log/aciplugin_logs
+   cp PluginCiscoACI/install/Kubernetes/helmcharts/aciplugin-config.yaml ~/plugins/aciplugin
```
Comment (sukhwinder20) [inline README.md:81]: change  cp PluginCiscoACI/install/Kubernetes/helmcharts/aciplugin-config.yaml ~/plugins/aciplugin to  cp ~/PluginCiscoACI/install/Kubernetes/helmcharts/aciplugin-config.yaml ~/plugins/aciplugin

File: README.md [hunk 3]
```diff
@@ -39,44 +52,48 @@ Kubernetes cluster is set up and the resource aggregator is successfully deploye
...
    ```
 
-5. On the deployment node, copy the Cisco ACI plugin configuration file to `~/plugins/aciplugin`.
-
    ```
-   $ cp ~/ODIM/odim-controller/helmcharts/aciplugin/aciplugin-config.yaml ~/plugins/aciplugin
+   cp PluginCiscoACI/install/Kubernetes/helmcharts/aciplugin.sh ~/plugins/aciplugin
```
Comment (sukhwinder20) [inline README.md:85]: change  cp PluginCiscoACI/install/Kubernetes/helmcharts/aciplugin.sh ~/plugins/aciplugin to  cp ~/PluginCiscoACI/install/Kubernetes/helmcharts/aciplugin.sh ~/plugins/aciplugin

File: README.md [hunk 4]
```diff
@@ -132,45 +151,45 @@ Kubernetes cluster is set up and the resource aggregator is successfully deploye
...
-9. Generate the Helm package for the Cisco ACI plugin on the deployment node:
 
-   1. Navigate to `odim-controller/helmcharts/aciplugin`.
+7. Generate the Helm package for the Cisco ACI plugin on the deployment node:
+
+   1. Navigate to `PluginCiscoACI/install/Kubernetes/helmcharts`.
 
       ```
-      $ cd ~/ODIM/odim-controller/helmcharts/aciplugin
+      cd PluginCiscoACI/install/Kubernetes/helmcharts
```
Comment (sukhwinder20) [inline README.md:162]: change cd PluginCiscoACI/install/Kubernetes/helmcharts to cd ~/PluginCiscoACI/install/Kubernetes/helmcharts

**Pull request merged**